src/main.c

```diff
@@ -107,32 +110,21 @@ static void select_callback(struct MenuLayer *menu_layer, MenuIndex *cell_index,
   }
 }
 
-static void draw_header_callback(GContext *ctx, const Layer *cell_layer, uint16_t section_index, void *context) {
-  menu_cell_basic_header_draw(ctx, cell_layer, "Choose a Component");
-}
-
-static int16_t get_header_height_callback(struct MenuLayer *menu_layer, uint16_t section_index, void *context) {
-  return MENU_CELL_BASIC_HEADER_HEIGHT;
-}
-
-static uint16_t get_num_sections_callback(struct MenuLayer *menu_layer, void *context) {
-  return 1;
-}
-
 static void window_load(Window *window) {
   Layer *window_layer = window_get_root_layer(window);
   GRect bounds = layer_get_bounds(window_layer);
 
   s_menu_layer = menu_layer_create(bounds);
   menu_layer_set_click_config_onto_window(s_menu_layer, window);
+  menu_layer_set_normal_colors(s_menu_layer, GColorBlack, GColorWhite);
+  menu_layer_set_highlight_colors(s_menu_layer, GColorRed, GColorWhite);
   menu_layer_set_callbacks(s_menu_layer, NULL, (MenuLayerCallbacks) {
       .get_num_rows = (MenuLayerGetNumberOfRowsInSectionsCallback)get_num_rows_callback,
```

Comment:
Should avoid casting these function pointers to a type in case the type is changed in the future.